.yamllint.yml

```
@@ -0,0 +1,38 @@
+---
+yaml-files:
+  - '*.yaml'
+  - '*.yml'
+  - .yamllint
+
+rules:
+  braces: enable
+  brackets: enable
+  colons: enable
+  commas: enable
+  comments:
+    level: warning
+  comments-indentation:
+    level: warning
+  document-end: disable
+  document-start:
+    level: warning
+  empty-lines: enable
+  empty-values: disable
+  float-values: disable
+  hyphens:
+    max-spaces-after: 4
```

```suggestion
    max-spaces-after: 3
```
I think this should be enough. My guess of what the parser is doing in these cases is that it will take 1 space after the hyphen of the outer list and two spaces for the inner list.